npf/Graph.py

```diff
@@ -0,0 +1,77 @@
+from npf.types.dataset import convert_to_xyeb
```

Comment:
In python filenames are lowercase.

Reply:
Git didn't pick up on the Graph.py to graph.py renaming, so I'll force it.